lib/ReactViews/Map/Panels/setting-panel.scss

```diff
@@ -26,6 +26,9 @@
 .btn--viewer {
   composes: btn from '../../../Sass/common/_buttons.scss';
   composes: btn-grey from '../../../Sass/common/_buttons.scss';
+
+  color: $text-light;
```

Comment:
Just wondering if it's worth adding a "deprecated" comment to these added lines as well, so we know we can safely remove them again one day?